WooCommerce/src/test/kotlin/com/woocommerce/android/ui/orders/wooshippinglabels/split/WooShippingSplitShipmentViewModelTest.kt

```diff
@@ -312,12 +312,70 @@ class WooShippingSplitShipmentViewModelTest : BaseUnitTest() {
 
         val state = sut.viewState.value!!
 
-        val selectableItems = state.selectableItems.getValue(1)
+        val selectableItems = state.selectableItems.getValue(0)
         assertThat(selectableItems.totalItemQuantity).isEqualTo(expectedQuantity)
     }
 
+    @Test
+    fun `when removing a shipment, then the total shipment count is reduced by 1`() = testBlocking {
+        val shipmentArgs = SplitShipmentArgs(
+            orderId = 1L,
+            storeOptions = StoreOptionsModel.EMPTY,
+            shipments = twoShipment
+        )
+        val movingItems = twoShipment.getValue(1)
+
+        // Removing "Shipment 2"
+        val movement = SplitMovement(
+            sourceShipmentKey = 1,
+            updatedSourceShipmentItems = emptyList(),
+            destinationShipmentKey = 0,
+            movingShipmentItems = movingItems
+        )
+
+        createViewModel(shipmentArgs)
+
+        sut.onUpdateShipment(movement)
+
+        sut.viewState.observeForTesting { }
+
+        val state = sut.viewState.value!!
+
+        assertThat(state.selectableItems.size).isEqualTo(1)
+    }
+
+    @Test
+    fun `when removing a shipment, then all items are moved to destination shipment`() = testBlocking {
+        val shipmentArgs = SplitShipmentArgs(
+            orderId = 1L,
+            storeOptions = StoreOptionsModel.EMPTY,
+            shipments = twoShipment
+        )
+        val movingItems = twoShipment.getValue(1)
```

Comment:
Minor np. Map access to values can be used like:
```suggestion
        val movingItems = twoShipment[1]!!
```

Reply:
Done: 6b8ad5adc701c81590e3b251af7b0b4599b5e764